runtime/ddr/gcddr.cpp

```diff
@@ -115,18 +113,6 @@ DDR_MM_HeapRegionDescriptor::getRegionType()
 	return this->_ddrRegionType;
 }
 
-class DDR_MM_MemoryPoolHybrid : public MM_MemoryPoolHybrid
-{
-public:
-	const char * ddrHelper();
-};
-
-const char *
-DDR_MM_MemoryPoolHybrid::ddrHelper()
-{
-	return this->_typeId;
-}
```

Comment:
My only potential concern is if this was required to force the compiler to properly describe some ancestor class of `MM_MemoryPoolHybrid`. We'll need to pay close attention to any effects this has on AIX builds.

Reply:
All builds pass with no issues _(Axxon: /build_info.php?build_id=48521)_